mne/fixes.py

```diff
@@ -907,6 +907,64 @@ def get_spectrogram():
 
 
 ###############################################################################
+# Scipy trim_mean (for mne.time_frequency.psd_welch) needed for scipy < 0.12
```

Comment:
... and move it here instead of in `fixes` (it's not a copy but a different-functionality adaptation now)